index.html

```diff
@@ -0,0 +1,666 @@
+                                Subcription Business, +2</p>
+                        </li>
+
+                        <li>
+                            <p><i class="fas fa-tag"></i>
+                                Index ventures, +4</p>
+                        </li>
+                    </UL>
+
+                    <ul class="ul4">
+                        <li>
+                            <p>Subscription Business, +2</p>
+                        </li>
+                        <li>Index ventures, +4</li>
+                    </ul>
+
+
+                </div>
+            </div>
+
+
+            <!-- -------------------------------------------- -->
+            <div class="article-container-4">
+                <div class="article-container-1">
+
+                    <ul class="ul1">
+
+                        <li>
+                            <p>Encoded Genomics</p>
+                        </li>
+                    </ul>
+
+                    <ul class="ul2">
+    
+                        <li>
+                            <div class="company-logo">
+                                <img class="article-img" src="img/image.jpeg" alt="Company logo" />
+                            </div>
+                        </li>
+                        <li>
+                        <p>$135M</p>
+                        </li>
+                        <li>
+                            <p>SERIES D</p>
+                        </li>
+                    </ul>
+
+
+
+                    <UL class="ul3">
+                        <li>
+                            <p><i class="fas fa-tag"></i>
+                                Subcription Business, +2</p>
+                        </li>
+
+                        <li>
+                            <p><i class="fas fa-tag"></i>
+                                Index ventures, +4</p>
+                        </li>
+                    </UL>
+
+                    <ul class="ul4">
+                        <li>
+                            <p>Subscription Business, +2</p>
+                        </li>
+                        <li>Index ventures, +4</li>
+                    </ul>
+
+
+                </div>
+            </div>
+
+        </div>
+
+        <!-- -----------------------categories---------------------------- -->
+
+        <section class=" more-list">
+            <div class="more-articles">
+                <p class="orange-headers">Hard Fork <i class="fas fa-chevron-right"></i></p>
+                <img class="header-img" src="img/latest-deals.png" alt="latest deals">
+                <div class="more-info-header">
+                    <div>
+                        <p class="orange thin">
+                            LATEST
+                        </p>
+                        <p class="orange">
+                            How your company can limit its liability for US data collection lawsuits
+                        </p>
+                    </div>
```

Comment:
- [ ] In all screen sizes, you are missing the text inside the image.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88701057-e666c280-d0df-11ea-8add-80eed6219895.png) | ![image](https://user-images.githubusercontent.com/61479648/88701091-f2528480-d0df-11ea-8103-83cea9a3b884.png) |

- [ ] The alignment seems off.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88701342-46f5ff80-d0e0-11ea-8ab9-4625df88aa69.png) | ![image](https://user-images.githubusercontent.com/61479648/88701374-5117fe00-d0e0-11ea-87bc-7ba3f4345637.png) |